more/pixels/pixelMintKeeper.js

```diff
@@ -77,36 +76,27 @@ export function makePixelMintKeeper(assay) {
     // that it is currently in. Destroy is outside of an assetKeeper
     // because it could affect purses *or* payments
     destroy(amount) {
-      // amount must only contain a pixelList of length 1 for now
       const pixelList = assay.quantity(amount);
-      insist(
-        pixelList.length === 1,
-      )`amount must contain a pixelList of length 1 for now`;
+      pixelList.forEach(pixel => {
+        const strPixel = getString(pixel);
+        if (pixelToAsset.has(strPixel)) {
+          const asset = pixelToAsset.get(strPixel);
+          amount = assay.coerce(amount);
```

Comment:
Please don't reassign to parameters.

Reply:
I think this is outside the scope of this PR since this PR didn't add these lines, and it's an ongoing discussion that we need to have. I would agree with you, though